libmamba/include/mamba/fs/filesystem.hpp

```diff
@@ -303,6 +306,12 @@ namespace mamba::fs
             return to_utf8(m_path);
         }
 
+        // Returns a default encoded string.
+        const std::string& native() const
```

Comment:
@Klaim I needed to add this function for pybind11, as it calls it and does utf-8 translation itself.
Is that OK?

Reply:
As long as it does what the standard does, that is returning the real native encoding of that the path use on that system, yes it's OK

I believe we didnt set this function to discourage trying to use something else than utf-8 with our library, but it's still just reflecting what the sandard does otherwise.